docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,93 @@
-# Versioning support
-
+# Versioning Support
 There are different components that might require independent versioning and support level:
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster. However, different versions of the HO are thoroughly tested only on a limited set of OpenShift versions, and this should inform your deployment decisions.
+
+For a given build of the upstream HO (e.g. [v0.1.39](https://github.com/openshift/hypershift/releases/tag/v0.1.39)), the list of OCP versions tested for Management Cluster compatibility can be found here: `link to file in repository with this information -- ideally where it is programmatically checked and results in warnings if it is not met at runtime`.
+
+For production use & support, it is required to use a downstream product which bundles a supported build of the HyperShift Operator. This downstream product is called [Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.16/architecture/mce-overview-ocp.html) (MCE) and it is available through OpenShift's OperatorHub. 
+
+MCE versions _do_ require specific OCP versions for the Management Cluster to remain in a supported state. 
+Each version documents its own support matrix. For example, 
 
-## Support level
+- [MCE 2.5](https://access.redhat.com/articles/7056007)
+- [MCE 2.4](https://access.redhat.com/articles/7027079)
+
+As a heuristic, a new release of MCE will run on:
+
+- The latest, yet to be released version of OpenShift
+- The latest GA version of OpenShift
+- Two versions prior to the latest GA version
+
+Versions of MCE can also be obtained with the Advanced Cluster Management (ACM) offering. If you are running ACM, refer to product documentation to determine the bundled MCE version.
+
+The full list of HostedCluster OCP versions that can be installed via the HO on a Management Cluster will depend on the version of the installed HO. However, if you are running a tested configuration or MCE, this list will always include at least (a) the same OCP version as the Management Cluster and (b) Two previous minor versions relative to the Management Cluster. For example, if the Management Cluster is running 4.16 and a supported version of MCE, then the HO will at least be able to install 4.16, 4.15, and 4.14 Hosted Clusters.
```

Comment:
Another thing that we can clarify here is that the management cluster's OCP version itself does not restrict hosted clusters' OCP version that HO can install or upgrade. There is no relationship or interaction between the management cluster's OCP version and hosted cluster's OCP version.